tests/unit/testConfigs/locConstraintAssert.js

```diff
@@ -0,0 +1,118 @@
+import assert from 'assert';
+import { locationConstraintAssert } from '../../../lib/Config';
+
+function generateConfig(testKey) {
```

Comment:
as discussed, create a default config and then here use object assign to create a new object and switch out the item you want to change.